packages/twenty-server/src/core/workspace/workspace.entity.ts

```diff
@@ -55,11 +55,6 @@ export class Workspace {
   @OneToMany(() => User, (user) => user.defaultWorkspace)
   users: User[];
 
-  @OneToMany(() => UserWorkspace, (userWorkspace) => userWorkspace.workspace, {
-    onDelete: 'CASCADE',
-  })
-  workspaceUsers: UserWorkspace[];
-
```

Comment:
think the OneToMany with userWorkspace entities is missing, but we might not call that field `users` as it is already taken for the OneToMany with user entities
Let's call it workspaceUsers